src/www/system_certmanager.php

```diff
@@ -1087,9 +1108,41 @@ function removeRowAltNm() {
         $("#certmethod").change();
     }
   });
+
+  function refresh_download_link(jquery_key, content, filename) {
+      if (navigator.userAgent.indexOf('MSIE ') !== -1 || navigator.userAgent.indexOf('Trident') !== -1) {
+          // IE11 support; they do not have <a download="">
```

Comment:
I'm not sure we should go through all that trouble trying to support IE11, what if we just make sure the user can copy-paste the content and optionally save to file if supported by the browser?

Reply:
Okay, I will remove download link if unsupported.